lib/Cake/Model/Datasource/DboSource.php

```diff
@@ -367,6 +367,15 @@ public function value($data, $column = null, $null = true) {
 			$column = $this->introspectType($data);
 		}
 
+		if (strpos($column, "enum")===0){
+			preg_match("/(enum\()(.*)(\))/",$column, $acceptingValues);
+			$firstValue = explode(",", $acceptingValues[2])[0];
```

Comment:
This is invalid syntax in PHP5.3 [a build failed because of this](https://travis-ci.org/cakephp/cakephp/jobs/452931595).